Common/src/grid_movement_structure.cpp

```diff
@@ -7755,7 +7815,16 @@ void CSurfaceMovement::WriteFFDInfo(CGeometry *geometry, CConfig *config) {
       output_file << "FFD_DEGREE_I= " << FFDBox[iFFDBox]->GetlOrder()-1 << endl;
       output_file << "FFD_DEGREE_J= " << FFDBox[iFFDBox]->GetmOrder()-1 << endl;
       if (nDim == 3) output_file << "FFD_DEGREE_K= " << FFDBox[iFFDBox]->GetnOrder()-1 << endl;
-      
+      if (config->GetFFD_Blending() == BSPLINE_UNIFORM) {
```

Comment:
At the moment it is not backward compatible .. it is assumed that the line FFD_BLENDING= is present in the mesh file. But I can change that.

As with all the FFD settings in the mesh file, there is no error check because normally there is no reason to manually modify these things. Of course there are default values set if you generate the FFD box.